tools/java/runfiles/Runfiles.java

```diff
@@ -225,6 +420,18 @@ private static String findRunfilesDir(String manifest) {
       return "";
     }
 
+    private static String findRepositoryMapping(String manifestPath) {
+      if (manifestPath != null && (manifestPath.endsWith(".runfiles/MANIFEST")
+          || manifestPath.endsWith(".runfiles\\MANIFEST")
+          || manifestPath.endsWith(".runfiles_manifest"))) {
+        String path = manifestPath.substring(0, manifestPath.length() - 18) + ".repo_mapping";
```

Comment:
Just a comment, nothing to do:`- 18` is "clever". Possibly causing problems in the future. But I see same thing is used elsewhere.